docs/resources/integration_logging.md

```diff
@@ -0,0 +1,107 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cyral_integration_logging Resource - cyral"
+subcategory: ""
+description: |-
+  Manages a logging integration that can be used to push logs from Cyral to the corresponding logging system (E.g.: AWS CloudWatch, Splunk, SumoLogic, etc).
+---
+
+# cyral_integration_logging (Resource)
+
+Manages a logging integration that can be used to push logs from Cyral to the corresponding logging system (E.g.: AWS CloudWatch, Splunk, SumoLogic, etc).
+
+<!-- schema generated by tfplugindocs -->
+
+## Schema
+
+### Required
+
+- `name` (String) Name of the logging integration config.
+
+### Optional
+
+- `cloud_watch` (Block Set, Max: 1) Represents the configuration data required for the `AWS` CloudWatch log management system. (see [below for nested schema](#nestedblock--cloud_watch))
+- `datadog` (Block Set, Max: 1) Represents the configuration data required for the Datadog's log management system. (see [below for nested schema](#nestedblock--datadog))
+- `elk` (Block Set, Max: 1) Represents the configuration data required for the ELK stack log management system. (see [below for nested schema](#nestedblock--elk))
+- `fluentbit` (Block Set, Max: 1) Represents a custom Fluent Bit configuration which will be utilized by the sidecar's log shipper. (see [below for nested schema](#nestedblock--fluentbit))
+- `receive_audit_logs` (Boolean) Whether or not Cyral audit logs should be forwarded to this logging integration.
+- `splunk` (Block Set, Max: 1) Represents the configuration data required for the Splunk log management system. (see [below for nested schema](#nestedblock--splunk))
+- `sumo_logic` (Block Set, Max: 1) Represents the configuration data required for the Sumo Logic log management system. (see [below for nested schema](#nestedblock--sumo_logic))
+
+### Read-Only
+
+- `id` (String) Unique identifier of the logging integration.
+
+<a id="nestedblock--cloud_watch"></a>
+
+### Nested Schema for `cloud_watch`
+
+Required:
+
+- `group` (String) CloudWatch log group.
+- `region` (String) AWS region.
+
+Optional:
+
+- `stream` (String) CloudWatch log stream.
```

Comment:
I think it defaults to `cyral-sidecar` if not provided. @ccampo133 , could you please confirm? @gabfelp , please add this info to the docs once we get the confirmation.

Reply:
Sure, updated the description accordingly: [35daafa](https://github.com/cyralinc/terraform-provider-cyral/pull/395/commits/35daafa7a278976ed6616a2964405590d94d3366)